ios/RNGoogleMobileAds/RNGoogleMobileAdsConsentModule.m

```diff
@@ -18,8 +18,17 @@
 
 #import <React/RCTUtils.h>
 
+#import "RCTBridgeModule.h"
 #import "RNGoogleMobileAdsConsentModule.h"
 #import "common/RNSharedUtils.h"
+#if __has_include(<React/RCTConvert.h>)
+#import <React/RCTConvert.h>
+#elif __has_include("RCTConvert.h")
+#import "RCTConvert.h"
+#else
+#import "React/RCTConvert.h"
+#endif
```

Comment:
I don't believe this has been necessary for a long time - I think they can just all be one-liner `#import <React/RCTConvert.h>` style headers now?
I seem to recall a PR taking this conditional import stuff *out* of some other module recently even, where it was commented that it was an artifact of some long long gone react-native version